### PR TITLE
PE 88 rocketchat notifications

### PR DESCRIPTION
**Pull request by andrey-canon**
## Description
This branch allows to get notifications on the rocketchat tab for this the following main files have been changed or added:
- rocketchat_notifications.js = Allows to use RocketChat realtime Api in order to get the new messages
- api/v0/views.py = Provides the credentials and room ids to the javascript.
- api/v0/serializers.py =  Returns the data to use on the response 

## Reviewers 
- [ ] @diegomillan 
- [ ] @jfavellar90

### CODE REVIEW

**Comment by nicovanniekerk** [comment]
LGTM

File: common/djangoapps/rocket_chat/api/v0/urls.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+"""
+API v1 URLs.
```
Comment (diegomillan) [inline common/djangoapps/rocket_chat/api/v0/urls.py]: I guess this should be `API v0 URLs.`

File: common/djangoapps/rocket_chat/api/v0/views.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+
+class RocketChatCredentials(APIView):
+
+    authentication_classes = (
+        SessionAuthentication,
+    )
+    permission_classes = (permissions.IsAuthenticated,)
+
+    def get(self, request, course_id):
```
Comment (diegomillan) [inline common/djangoapps/rocket_chat/api/v0/views.py:29]: Please let's add a docstring for every API method/view

File: common/djangoapps/rocket_chat/api/v0/views.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        rocket_chat_settings = get_rocket_chat_settings()
+
+        api_rocket_chat = initialize_api_rocket_chat(rocket_chat_settings)
+
+        if api_rocket_chat:
+
+            user_info = api_rocket_chat.users_info(username=user.username)
+
+            try:
+                user_info = user_info.json()
```
Comment (diegomillan) [inline common/djangoapps/rocket_chat/api/v0/views.py:46]: Let's try to call user creation once

**Review by diegomillan — CHANGES_REQUESTED** [review]

**Review by diegomillan — APPROVED** [review]

**Pull request merged**